CHANGELOG.md

```diff
@@ -5,14 +5,21 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
+- Execute SSH key creation alongside with the creation of HOME directory, i.e.
+  during SSH login, when switching to another user and when executing a command as another user.
+- Add support for both FQDN and LDAP Distinguished Names in property `DirectoryService.DomainName`.
+- Add script in the head node to manually update the password used to read from Active Directory, 
```

Comment:
New `<xxx>` script deployed on the head node supports the manual update of the Active Directory password in the SSSD configuration. The password is retrieved by the AWS Secret Manager as from the cluster configuration.